mobile/lib/repositories/asset_api.repository.dart

```diff
@@ -22,4 +26,26 @@ class AssetApiRepository extends BaseApiRepository
     );
     return Asset.remote(response);
   }
+
+  @override
+  Future<List<Asset>> search({List<String> personIds = const []}) async {
+    final List<Asset> result = [];
+    bool hasNext = true;
+    int currentPage = 1;
+    while (hasNext) {
```

Comment:
Maybe we should add a to-do to make the mobile implementation use pagination, and not fetch all the results all the time.